example_dags/astroflow_render_example.py

```diff
@@ -0,0 +1,58 @@
+import os
```

Comment:
What are your thoughts on naming this as `example_snowflake_render.py`, so we follow the current naming convention?